docs/dns/rrsets.rst

```diff
@@ -213,6 +213,20 @@ https://www.huque.com/bin/gen_tlsa.  We are planning to provide a tool that is
 connected directly to our API in the future.  For full detail on how ``TLSA``
 records work, please refer to RFC 6698.
 
+The ``TLSA`` sha256 hash can also be generated with that command::
+
+    openssl x509 -in /etc/letsencrypt/live/${name}/fullchain.pem -noout -pubkey \
+    | openssl pkey -pubin -outform DER \
+    | openssl sha256
+
+**Pro tip:** To put only the hash into an environment variable can this comand
```

Comment:
Spelling: command

Grammar: this command can be used
And I think a comma should precede this phrase (but I'm not 100% sure about that).